packages/curve-ui-kit/src/shared/ui/SliderInput.tsx

```diff
@@ -0,0 +1,197 @@
+import { TextFieldProps } from '@mui/material'
+import Stack from '@mui/material/Stack'
+import { SliderSize } from '@ui-kit/themes/components/slider/types'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { type Decimal } from '@ui-kit/utils'
+import { NumericTextField, NumericTextFieldProps } from './NumericTextField'
+import { Slider, type SliderProps } from './Slider'
+
+const { Spacing, MaxWidth } = SizesAndSpaces
+
+export type RangeValue = [number, number]
+
+export type SliderInputProps = {
+  /** The aria-label */
+  ariaLabel?: string
+  /** The direction of the layout. Row: inputs on the left and right of the slider. Column: inputs below the slider. */
+  layoutDirection?: 'column' | 'row'
```

Comment:
If you decide to merge this component with `Slider`, we can probably also turn this into `orientation: MuiSliderProps['orientation']`?